release-notes.md

```diff
@@ -204,4 +204,7 @@
 * `.code.text` ; mapping ; Moved mapping of FurtherSpecificationProblemName from `.code.extension:furtherSpecificationProblemName.value[x]`. ; ZIBFHIR-145
 * `.bodySite` ; cardinality ; Changed max cardinality from 1 to *. ; ZIB-1433
 * `.bodySite` ; terminology ; Binding strength changed from required to extensible. ; ZIBFHIR-113
-* `.note.text` ; textual ; Changed definition of Comment. ; ZIB-1610
\ No newline at end of file
+* `.note.text` ; textual ; Changed definition of Comment. ; ZIB-1610
+
+## SNAQScore (zib-SNAQScore, nl-core-SNAQScore)
+* `.component:weightLossScore.value[x]` ; terminology ; Changed codes in WeightLossScoreCodelist. ; ZIB-1729
```

Comment:
This is not true. The change was already present in the zib2020 FHIR implementation by means of an erratum. The definition text of Observation.valueInteger did change though, but I think that's actually an omission in zib2020 where the erratum wasn't applied to the definition text. Can you make a ZIBFHIR change request for that and change this line to "No significant changes"?